README.md

```diff
@@ -14,7 +14,7 @@ comprehensive set of [common tools](https://godotengine.org/features), so that u
 without having to reinvent the wheel. Games can be exported with one click to a
 number of platforms, including the major desktop platforms (Linux, macOS,
 Windows), mobile platforms (Android, iOS), as well as Web-based platforms
-(HTML5) and
+(Web) and
```

Comment:
Maybe the whole parenthesis can be removed, it's kind of redundant now. And the `and [consoles](...)` can likely be put on a single line.